Jenkinsfile

```diff
@@ -43,6 +44,9 @@ pipeline {
             excludeMessage('The system property test.engine.url is configured twice!.*')
           ]
           junit '**/**/target/*-reports/**/*.xml'
+
+          archiveArtifacts '**/**/target/testEngineOut.log'
+          archiveArtifacts '**/**/target/failsafe-reports/**/*'

```

Comment:
more feedback on the last run .... would have helped to see that not all builds fail .

Reply:
Why am I on this PR? Do you wanted to add @ivy-cst as reviewer?

Reply:
Because you introduced the breaking change: https://github.com/axonivy/project-build-examples/commit/075e4f6
And ivy-cst because we investigated onto these problems, but didn't find the cause at first.